src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_updateTier.java

```diff
@@ -56,6 +57,28 @@ public void createsContactWhenTierUpdated() {
         List<Contact> updatedContacts = contactService.contactsFor(2500343964L, contactFilter);
 
         assertThat(updatedContacts.stream().anyMatch(c ->  c.getNotes().contains("Tier: B-1"))).isTrue();
+        managementTierRepository.deleteById(tierB1.getId());
+    }
+
+
```

Comment:
another test to bring in a tier when it is the first tier calculation.

Reply:
added a test for it, you were correct, null was possible